views/marts/product/fct_active_users.view.lkml

```diff
@@ -123,28 +178,84 @@ view: fct_active_users {
   dimension: weekly_desktop_active_users {
     type: number
     sql: ${TABLE}.weekly_desktop_active_users ;;
-    label: "Desktop/Client WAU"
-    description: "WAU reported from Rudderstack"
+    label: "Desktop WAU"
+    description: "Desktop WAU reported by telemetry."
   }
 
   measure: total_weekly_desktop_active_users {
     type: sum
     sql: ${weekly_desktop_active_users} ;;
-    label: "Total Desktop/Client WAU"
+    label: "Total Desktop WAU"
+    view_label: "  * Metrics. WAU"
+  }
+
+  dimension: weekly_webapp_active_users {
+    type: number
+    sql: ${TABLE}.weekly_webapp_active_users ;;
+    label: "Webapp WAU"
+    description: "Webapp WAU reported by telemetry."
+  }
+
+  measure: total_weekly_webapp_active_users {
+    type: sum
+    sql: ${weekly_webapp_active_users} ;;
+    label: "Total Webapp WAU"
+    view_label: "  * Metrics. WAU"
+  }
+
+  dimension: weekly_client_desktop_active_users {
+    type: number
+    sql: ${TABLE}.weekly_client_desktop_active_users ;;
+    label: "Client Desktop WAU"
+    description: "Client WAU reported by Rudderstack telemetry."
```

Comment:
"Client" -> "Client Desktop" or "Desktop"?

Reply:
This is now updated.